builds/assets/lang/enAU.json

```diff
@@ -149,6 +149,8 @@
   "settingsVisContoursColour": "Contours Colour",
   "settingsVisContoursDistances": "Contours Distances",
   "settingsVisContoursBandHeight": "Contours Band Height",
+  "settingsVisContoursRainboxRepeatRate": "Rainbox Repeat Distance",
+  "settingsVisContoursRainboxIntensity": "Rainbox Intensity",
```

Comment:
`Rainbox`?

Reply:
That's a typo that is apparently everywhere; will correct.